sources/packages/web/src/views/aest/student/AssessmentAward.vue

```diff
@@ -79,11 +91,30 @@ export default defineComponent({
       });
     };
 
+    const confirmEnrolment = async (disbursementId: number) => {
+      try {
+        await ConfirmationOfEnrollmentService.shared.confirmEnrollment(
+          disbursementId,
+        );
+        snackBar.success("Confirmation of Enrollment Confirmed!");
+        loadAssessmentAwardValues();
+      } catch (error: unknown) {
+        if (
+          error instanceof ApiProcessError &&
+          error.errorType === FIRST_COE_NOT_COMPLETE
+        ) {
+          snackBar.error("First COE is not completed.");
+        }
+        snackBar.error("An error happened while confirming the COE.");
```

Comment:
Doesn't it require an else here?

Reply:
Yes you are right. I can do a return right after line 104.